modules/op-release-notes-1-2.adoc

```diff
@@ -0,0 +1,107 @@
+* You can now use the `--prefix-name` option with the `tkn clustertask start` command to specify a prefix for the `TaskRun` name.
+* Interactive mode support has now been provided to the `tkn clustertask start` command.
+* You can now specify `PodTemplate` properties supported by pipelines using local or remote file definitions for `TaskRun` and `PipelineRun` objects.
+* You can now use the `--use-params-defaults` option with the `tkn clustertask start` command to use the default values set in the `ClusterTask` configuration and create the `TaskRun`.
+* The `--use-param-defaults` flag for the `tkn pipeline start` command now prompts the interactive mode if the default values have not been specified for some of the parameters.
+
+=== Triggers
+
+* The Common Expression Language (CEL) function named `parseYAML` has been added to parse a YAML string into a map of strings.
+* Error messages for parsing CEL expressions have been improved to make them more granular while evaluating expressions and when parsing the hook body for creating the evaluation environment.
+* Support is now available for marshaling boolean values and maps if they are used as the values of expressions in a CEL overlay mechanism.
+* The following fields have been added to the `EventListener` object:
+** The `replicas` field enables the event listener to run more than one pod by specifying the number of replicas in the YAML file.
+** The `NodeSelector` field enables the `EventListener` object to schedule the event listener pod to a specific node.
+* Webhook interceptors can now parse the incoming `EventListener-Request-URL` to extract parameters from the original request URL being handled by the `EventListener` object.
+* Annotations from the event listener can now be propagated to the deployment, services, and other pods. Note that custom annotations on services or deployment are overwritten, and hence, must be added to the event listener annotations so that they are propagated.
+* Proper validation for replicas in the `EventListener` specification is now available for cases when a user specifies the `spec.replicas` values as `negative` or `zero`.
+* You can now specify the `TriggerCRD` object inside the `EventListener` spec as a reference using the `TriggerRef` field to create the `TriggerCRD` separately and then bind it inside the `EventListener` spec.
+* Validation and defaults for the `TriggerCRD` object are now available.
+
+[id="deprecated-features-1-2_{context}"]
+== Deprecated features
+
+* `$(params)` are now deprecated and replaced by `$(tt.params)` to avoid confusion between the `resourcetemplate` and `triggertemplate` parameters.
+* The `ServiceAccount` reference of the optional `EventListenerTrigger`-based authentication level has changed from an object reference to a string `ServiceAccountName`. This ensures that the `ServiceAccount` is in the same namespace as the `EventListenerTrigger`.
```

Comment:
s/to a string `ServiceAccountName`./to a `ServiceAccountName` string.

Reply:
s/`ServiceAccount` is in/`ServiceAccount` reference is in/
s/as the `EventListenerTrigger`./as the `EventListenerTrigger` object./